js/coinbase.js

```diff
@@ -279,54 +288,37 @@ module.exports = class coinbase extends Exchange {
 
     async fetchMySells (symbol = undefined, since = undefined, limit = undefined, params = {}) {
         // they don't have an endpoint for all historical trades
-        const accountId = this.safeString2 (params, 'account_id', 'accountId');
-        if (accountId === undefined) {
-            throw new ArgumentsRequired (this.id + ' fetchMyTrades requires an account_id or accountId extra parameter, use fetchAccounts or loadAccounts to get ids of all your accounts.');
-        }
+        const request = await this.prepareAccountRequest (symbol, limit, params);
```

Comment:
Unfortunately, we can't use `prepareAccountRequest` interchangeably for both symbols and currency codes, so, in this case, we might need to revert it.

Reply:
The same is true for `fetchMyBuys` below.

Reply:
Of course.  Thanks

Reply:
Have made an amendment